stories/adminbar.stories.js

```diff
@@ -87,4 +90,34 @@ storiesOf( 'Global', module )
 				</div>
 			</div>
 		);
+	} )
+	.add( 'Admin Bar Zero Data', () => {
+		global._googlesitekitLegacyData = wpAdminBarData;
+
+		const setupRegistry = ( registry ) => {
+			// Set some site information.
+			provideSiteInfo( registry, { currentEntityURL: '/example-page/' } );
+
+			// Set up the search console and analytics modules stores but provide no data.
+			provideModules( registry, [ { slug: 'search-console', active: true, connected: true } ] );
+			provideModules( registry, [ { slug: 'analytics', active: true, connected: true } ] );
```

Comment:
This should be combined into a single call to `provideModules` since it is not additive, but the latter will overwrite the former.

```suggestion
			provideModules( registry, [
				{ slug: 'search-console', active: true, connected: true },
				{ slug: 'analytics', active: true, connected: true },
			] );
```